recipes/arrow/all/conanfile.py

```diff
@@ -26,6 +26,7 @@ class ArrowConan(ConanFile):
         "encryption": [True, False],
         "filesystem_layer":  [True, False],
         "hdfs_bridgs": [True, False],
+        "runtime_simd_level": [None, "sse4_2", "avx2", "avx512", "max"],
```

Comment:
We have been very inconsistent with this.... it seem the most common is 

```suggestion
        "simd": [None, "sse4_2", "avx2", "avx512", "max"],
```

Can we doc this or maybe add a hook so consumer lives are easier?

Reply:
yes, sounds better. @uilianries please apply and we can move forward.
also please think about `max` value, which is essentially `auto`, and makes unreproducible builds

Reply:
@SSE4 applied :)